Cargo.toml

```diff
@@ -13,3 +13,4 @@ mime-sniffer = "0.1.2"
 regex = "1.2.1"
 reqwest = "0.9.20"
 url = "2.1.0"
+lazy_static = "1.3.0"
```

Comment:
could you please add a newline at end of file here?

Reply:
yes sure